examples/rust-dataflow/dataflow.yml

```diff
@@ -7,11 +7,10 @@ nodes:
         tick: dora/timer/millis/10
       outputs:
         - random
-  - id: runtime-node
-    operators:
-      - id: rust-operator
+  - id: rust-operator
```

Comment:
Could we rename the node to something else? The term "operator" is probably confusing since this is no longer an operator.

Please also rename the executable and the node name in the log outputs. Thanks!

Reply:
Done: switched to `rust-status-node`